.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "ThirdParty/vtkGUISupportQt"]
 	path = ThirdParty/vtkGUISupportQt
 	url = https://github.com/ufz/vtkGUISupportQt.git
+[submodule "ThirdParty/metis"]
+	path = ThirdParty/metis
+	url = https://github.com/envinf/metis.git
```

Comment:
Not sure about putting metis into 'envinf'-github. Are there any changes we do to the source code of metis which should be closed? If not I'd think taking the origin of metis for downloading it is better.
In any case making a clone into 'ufz'-github would be better.